blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_5_structural_analysis/formula_5_3b.py

```diff
@@ -0,0 +1,66 @@
+"""Formula 5.3b from NEN-EN 1992-1-1+C2:2011: Chapter 5 - Structural Analysis."""
+
+from blueprints.codes.eurocode.nen_en_1992_1_1_c2_2011 import NEN_EN_1992_1_1_C2_2011
+from blueprints.codes.formula import Formula
+from blueprints.codes.latex_formula import LatexFormula
+from blueprints.type_alias import DIMENSIONLESS, KN
+from blueprints.validations import raise_if_negative
+
+
+class Form5Dot3BTransverseForceBracedMembers(Formula):
```

Comment:
@egarciamendez 
I have discussed this with Wichard and changed the suffix notation to lower case: Form5Dot3B -> Form5Dot3b. As this was not consistent with some other formulas that have already been implemented, I have changed the notation there as well [Better change 10 files now than regret not changing it for 100 files later 😉]. 
I have not changed the notation for formulas with suffix "N". These come from the national annex and have suffix "N" (uppercase) in the Eurocode. Since I can't recall what we do with the national annex, I would like to ask you: What's our policy towards the national annex? And should I also change the uppercase "N" suffix to lowercase "n" suffix? Please let me know!

Reply:
Hi guys, I apologize for the delayed response. The last few days have been quite busy for me. Regarding this matter, I don't hold a strong opinion. Altering "N" seems a bit unnecessary to me, but I'm fine with it if you all decide to proceed.